.github/workflows/cleanup-resources.yaml

```diff
@@ -0,0 +1,13 @@
+name: Cleanup dangling Azure resources
+on:
+  schedule:
+    - cron: "*/15 * * * *"
```

Comment:
Do you think every 15min is the right cadence (assuming my cron syntax is correct). It feels like it might be unnecessarily frequent for a cleanup job that hopefully won't have much to do too often

Reply:
The job would be triggered every 15 mins but would only actually check resources that were created something like >=2 hours ago, as done below. That way resources stay alive at most 2h15min.

https://github.com/jepio/kata-azure-automation/blob/a4113f68ca31da5f36afda0ddac85db56810cfd4/autodelete-aks/function_app.py#L36

Reply:
Please, make this running every 2 hours, if you want to be really cautious about the resources leaking.  But every 15 minutes is a bit excessive, IMHO.

Reply:
Changed it to every 6 hours so that the GHA UI doesn't accumulate more than 4 runs per day.

Reply:
Thanks!